includes/functions.php

```diff
@@ -4338,3 +4338,106 @@ function dokan_bool_to_on_off( $bool ) {
 
     return true === $bool ? 'on' : 'off';
 }
+
+/**
+ * Get WC orders after filtration
+ *
+ * @since DOKAN_SINCE
+ *
+ * @param array $args
+ *
+ * @return array
+ */
+function dokan_get_orders( $args_parameter ) {
```

Comment:
@NAbdulla1 kindly use dokan()->order->all method to get  order data, no need to introduce a new method

Reply:
vai, for dokan()->order->all we must provide a seller id otherwise if we send empty as seller id we get empty result, if we send nothing then default will be current user id as seller id. that's why a another method is introduced.